process-cli/pom.xml

```diff
@@ -9,6 +9,276 @@
   <artifactId>sonarlint-process-cli</artifactId>
   <name>SonarLint Core - Process CLI</name>
   <description>Entry point for SonarLint process</description>
+  <properties>
+    <unpack.dir>${project.build.directory}/unpack</unpack.dir>
+    <jre.dirname.linux>jdk-17.0.7+7-jre</jre.dirname.linux>
+    <jre.dirname.windows>jdk-17.0.7+7-jre</jre.dirname.windows>
+    <jre.dirname.macosx>jdk-17.0.7+7-jre/Contents/Home</jre.dirname.macosx>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>org.sonarsource.sonarlint.core.rpc.impl.SonarLintCoreProcess</mainClass>
+              <addClasspath>true</addClasspath>
+            </manifest>
+          </archive>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+      <profile>
+        <id>dist-windows</id>
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.6.8</version>  <!--Issue in 1.7.1 https://github.com/maven-download-plugin/maven-download-plugin/issues/260 -->
```

Comment:
Maybe use a property to avoid repeating the version

Reply:
Addressed in [79ddfa3](https://github.com/SonarSource/sonarlint-core/pull/757/commits/79ddfa394bd9a8f78543ac932e2eda5e69cdc53b)